mitosheet/src/mito/components/modals/UserEditedCodeModal.tsx

```diff
@@ -37,7 +40,33 @@ const UserEditedCodeModal = (
                         variant='light'
                         width='hug-contents'
                         onClick={() => {
-                            props.onClickButton(true);
+                            props.jupyterUtils?.writeGeneratedCodeToCell(
```

Comment:
these two on clicks are the same could be a function probably -- just defined in the component. :)